.github/workflows/build.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         # Use the community bundle because it's smaller and faster
         git clone --recurse-submodules https://github.com/adafruit/CircuitPython_Community_Bundle.git
         cd CircuitPython_Community_Bundle
-        circuitpython-build-bundles --filename_prefix test-bundle --library_location libraries --library_depth 2
+        circuitpython-build-bundles --filename_prefix test-bundle --library_location libraries --library_depth 2 --skip_failed
```

Comment:
I added more notes to the related PR but I want to note here that I'm uneasy about adding skip_failed during the test of circuitpython-build-tools: If a bug is introduced into the code here, --skip_failed might cover it up.

Instead, consider using the communty bundle *at a specific ref/tag* that is known to build properly, and remove --skip_failed.

Reply:
What do you think about having this task use the main Adafruit Bundle instead of the community bundle? 

The comment here notes that community bundle was chosen because it's smaller and thus faster. It is still true that the community bundle is smaller, but it's also gaining ground at ~158 libraries right now. I think the main bundle is ~340ish but am not certain.  I imagine that the disparity was greater when this task was initially set up.

I am not so sure that the increased speed is that big of a benefit. As I understand it this is always run automatically inside of github actions, there isn't typically a person sitting and waiting for it to complete.

In my mind pointing it to the main Adafruit Bundle would also be nice because it will give another chance to discover ASAP if any changes to the build-tool could cause any issues for the Adafruit libraries or bundle.